ketcher-autotests/tests/utils/selectors/templateModal.ts

```diff
@@ -113,6 +114,22 @@ export async function selectSaltsAndSolvents(
   await expect(page.getByTestId('templates-modal')).toHaveCount(0);
 }
 
+export async function selectSaltsAndSolventsGroup(
```

Comment:
Why can't we use `selectSaltsAndSolvents` function instead?

Reply:
Done.